src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTypeCheck.java

```diff
@@ -459,6 +470,32 @@ private void checkTag(DetailAST ast, List<JavadocTag> tags, String tagName,
         }
     }
 
+    /**
+     * Verifies that a record definition has the specified param tag for
+     * the specified record component name.
+     *
+     * @param ast           the AST node for the record definition.
```

Comment:
nit: might remove the extra spacing for these two params, assuming that's the convention in the rest of the code

Reply:
I do not remember we discussed this.
single space definitely make it harder to read it as code(in html it is not a problem as extra formatting is applied).
I am not ready to discuss what is best separator length, we will get it empirically eventually. Ones new Check appear on this we will decide on convention.

Reply:
Fixed to reflect good example.

Reply:
That was BAD example, completely hard to read by humans, single space separator is BAD or there should be some rules on how description is starting.
But we will come back to this a bit later, special Check is required to be created for this.